src/App.tsx

```diff
@@ -1,42 +1,44 @@
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
 
+import { Tabs } from './components/Tabs';
+
 export const tabs = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
 export const App: React.FC = () => {
+  const [activeTab, setActiveTab] = useState(1);
+  const selectedTabId = (id: string) => +id.slice(-1);
+  const checkId = (id: string) => (
+    id !== tabs[0].id
+    || id !== tabs[1].id
+    || id !== tabs[2].id);
+
+  const handlerClick = (id: string) => {
```

Comment:
```suggestion
  const handleClick = (id: string) => {
```
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364